packages/block-library/src/cover/edit.js

```diff
@@ -309,7 +316,9 @@ function CoverEdit( {
 
 	const hasBackground = !! ( url || overlayColor.color || gradientValue );
 	const showFocalPointPicker =
-		isVideoBackground || ( isImageBackground && ! hasParallax );
+		isRepeated ||
```

Comment:
`isRepeated` makes sense for image only, no? Also it 'cancels' the previous logic about focal point below.

So depending on whether we want to show the `FocalPointPicker` on `repeated` images the below condition (`( isImageBackground && ! hasParallax )` should change to reflect the wanted behavior.

Reply:
> isRepeated makes sense for image only, no?

correct.

> Also it 'cancels' the previous logic about focal point below.

I'd say _updates_ ;-)

> ...to reflect the wanted behavior.

The wanted behavior is being able to set a repeated background, may set it as fixed, and why not change the focal point of it. That works for me.